Dockerfile

```diff
@@ -4,12 +4,14 @@
 # image minimal in size.
 
 # must be at the top to use it in FROM clauses
+ARG ENVIRONMENT_SETTINGS=production
```

Comment:
this is only needed for the backend-build stage, so I'd move it over there (with the default), close to the `pip install` line (around 42)